src/components/v5/frame/ColonyHome/partials/FundsCards/partials/FundsCardsSubTitle.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { type FC } from 'react';
+
+interface FundsCardsSubTitleProps {
+  value: React.ReactNode;
+  currency: string;
+}
+export const FundsCardsSubTitle: FC<FundsCardsSubTitleProps> = ({
+  value,
+  currency,
+}) => {
+  return (
+    <div className="flex items-center gap-2 heading-4">
+      <b>{value}</b>
```

Comment:
I think use of the `<b>` tag is discouraged. Could just wrap in a `<p>` tag with the span inside of it.